packages/itwinui-css/src/style/global.scss

```diff
@@ -35,31 +35,38 @@ html {
   }
 }
 
-.iui-body {
-  background-color: var(--iui-color-background-2);
+:where(.iui-root) {
   color: var(--iui-text-color);
   font-size: $iui-font-size;
   line-height: $iui-line-height;
 
-  @include iui-scrollbar;
   @include iui-font-family;
 
-  &.iui-no-webfonts {
-    font-family: $iui-sans-fallback;
+  // reset margin and apply background only for <body> element
+  @at-root body#{&} {
+    margin: 0;
```

Comment:
We were not resetting margin before or I missed something?
Then it is another breaking change and this can be felt in users apps.

Reply:
By default, the browser stylesheet adds 8px margin to `<body>` so this is resetting that. Previously we didn't have it because were not targeting body, so I guess this counts as a new feature.

In the next push I've reduced the specificity to 0 by adding `:where`, so if the user has a different margin in their app, it would still take preference.